src/components/PageWrapper.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const PageWrapper = ({ children, ...props }) => (
```

Comment:
As we're not using babel (and don't particularly want to at this stage) and the spread operator isn't support in IE11 (which is included in our target browser list) I don't think we can use this without some sort of shim. WDYT?

Reply:
We are using Babel, it comes with create-react-app, so don't worry. Without Babel the entire frontend would not work considering that a) JSX is transformed by Babel and not supported by a single browser and b) import/export syntax, or even require, is not supported by browsers without babel.

Reply:
Sorry should have bene more clear, I meant we aren't using babel explicitly. However, you're right the app wouldn't work without it anyway! In that case, we should probably just use the babel eslint plugin.